content/applications/finance/accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -499,22 +507,35 @@ Common Errors
 There are multiple reasons behind a rejection from the SII, but these are some of the common errors
 you might have and which is the related solution. 
 
-- Error: RECHAZO- DTE Sin Comuna Origen.
+- Error: ``RECHAZO- DTE Sin Comuna Origen.``
   
   *Hint:* Make sure the Company Address is properly filled including the State and City. 
 
-- Error en Monto :  - IVA debe declararse.
+- Error en Monto: ``- IVA debe declararse.``
   
   *Hint:* The invoice lines should include one VAT tax, make sure you add one on each invoice line. 
 
-- Error: Rut No Autorizado a Firmar.
+- Error: ``Rut No Autorizado a Firmar.``
   
   *Hint:* The invoice lines should include one VAT tax, make sure you add one on each invoice line. 
 
-- Error: Fecha/Número Resolucion Invalido RECHAZO- CAF Vencido : (Firma_DTE[AAAA-MM-DD] - 
-  CAF[AAAA-MM-DD]) &gt; 6 meses.
+- Error: ``Fecha/Número Resolucion Invalido RECHAZO- CAF Vencido : (Firma_DTE[AAAA-MM-DD] -
+  CAF[AAAA-MM-DD]) &gt; 6 meses.``
   
   *Hint:* Try to add a new CAF related to this document as the one you’re using is expired.
+  
+- Error: ``Element '{http://www.sii.cl/SiiDte%7DRutReceptor': This element is not expected. Expected
+  is ( {http://www.sii.cl/SiiDte%7DRutEnvia ).``
+  
+  *Hint:* Make sure the field Document Type and VAT are set either in the Customer and in the main
+  company.
+   
+- GLOSA: ``Usuario sin permiso de envio.``
```

Comment:
```suggestion
- Error: ``Usuario sin permiso de envio.``
```
Why GLOSA ? I'd expect the same structure than previous points and an english term outside of the error quote :thinking:?
(Sorry for yet another remark :D)

Reply:
Well, I wouldn't have thought it was a common noun. I thought it was an acronym or something ^^
It seems that it would rather mean "note", more than "error."

@ren-odoo any insight? :)

Reply:
There's also an "error en monto" label above.